R/interact.R

```diff
@@ -93,7 +93,7 @@ step_interact <-
     add_step(
       recipe,
       step_interact_new(
-        terms = terms,
+        terms = enquos(terms),
```

Comment:
This is the original cause of the problem, since `terms` is a required argument, it wouldn't let it be empty. turning it into quosures like the rest of the steps treats selectors seems like the right choice